SpatialGDK/Documentation/content/glossary.md

```diff
@@ -34,17 +31,14 @@ Short for “Interop Code Generator”.
 Short for “interoperable” and “interoperability”. See [Stack Overflow](https://stackoverflow.com/questions/5300383/interoperability).
 
 ### Interop Code Generator
```

Comment:
I've said this a few times but never with everyone. Could we rename this to Interop Schema Generator